doc/tntcxx_api.rst

```diff
@@ -28,141 +26,184 @@ Methods:
 
 .. _tntcxx_api_connector_connect:
 
-.. c:function:: int connect(Connection<BUFFER, NetProvider>& conn, const std::string_view& addr, unsigned port)
+..  c:function:: int connect(Connection<BUFFER, NetProvider>& conn, const std::string_view& addr, unsigned port)
 
-   Connects to a Tarantool instance that is listening on ``addr:port``.
+    Connects to a Tarantool instance. If a host doesn't reply within a timeout
+    period (2 seconds by default), an error is returned.
 
-   :param conn: connection object <cross-ref to corresp. topic>.
-   :param addr: URL of a host where a Tarantool instance is running.
-   :param port: a port that a Tarantool instance is listening to.
+    :param conn: connection object of the ``Connection`` class.
+    :param addr: address of the host where a Tarantool instance is running.
+    :param port: port that a Tarantool instance is listening to.
 
-   :return: connection status code.
-   :rtype: <TBD>
+    :return: ``0`` on success.
+    :return: ``-1`` otherwise. An error message is set that can be get by
+             the ``connection.getError()`` method.
```

Comment:
Using two `:return:` attributes seems wrong. There's one return value in the method, anyway.
Also, we can replace passive voice with active:
```suggestion
    :return: ``0`` on success, ``-1`` otherwise.
                 In case of an error, ``connection.getError()`` will give the error message.
```

Reply:
@Korablev77 `connection.getError()` возвращает сообщение об ошибке (строку) или объект/struct ошибки?

Reply:
std::string - наверное можно считать строкой?

Reply:
Re: two `:return:` attributes: agree, it can be confusing. We have such a template for describing functions in other places of our documentation (for example, https://www.tarantool.io/en/doc/latest/dev_guide/reference_capi/box/) but it seems we need to correct such places as well.